dart/deepgram_transcribe_video/README.md

```diff
@@ -0,0 +1,93 @@
+# 📹 Transcribe Video
+
+A Dart Cloud Function that transcribes pre-recorded video using [Deepgram](https://developers.deepgram.com/).
+
+ _Example input 1:_
+
+ ```json
+ {
+     "payload": "{\"fileUrl\": \"https://rawcdn.githack.com/deepgram-devs/transcribe-videos/62fc7769d6e2bf38e420ee5224060922af4546f7/deepgram.mp4\"}"
+ }
+ ```
+
+ _Example output 1:_
+
+ ```json
+ {
+     "success": true,
+     "deepgramData": { "A big": "Object" }
+ }
+ ```
+
+ _Example input 2:_
+ ```json
+ {
+     "payload": "{\"fileUrl\": \"Wrong\"}"
+ }
+ ```
+
+ _Example output 2:_
+ ```json
+ {
+     "success": false,
+     "message": "Failed to parse URL."
+ }
+ ```
+
+ _Example input 3:_
+
+ ```json
+ {
+      "payload": "{}"
+ }
+ ```
+
+ _Example output 3:_
+
+ ```json
+ {
+     "success": false,
+     "message": "Please provide a valid file URL in payload"
+ }
+ ```
+
+ If the API Key is not provided, you would get:
+
+ ```json
+ {
+     "success": false,
+     "message": "Provide the API key in environment variables"
+ }
+ ```
+
+ ## 📝 Environment Variables
+
+ List of environment variables used by this cloud function:
```

Comment:
Let's remove references to Environment since that was in older runtimes:

```suggestion
 ## 📝 Variables

 List of variables used by this cloud function:
```